lib/utils/job-utils/catalog-searcher.js

```diff
@@ -53,15 +53,15 @@ function searchCatalogDataFactory(
     }
 
     /**
-     * Get extended driveId catalog data with virtual disk info from megaraid-virtual-disks
+     * Get extended driveId catalog data
      * @param {String} nodeId - node identifier
      * @param {Object} filter - [optional] The filter which contains the driveId catalogs identifier
-     *                          Or devName. For example: {'sda': 1, 'sdb': 1, '3': 1}. 
+     *                          Or devName. For example: {'sda': 1, 'sdb': 1, '3': 1}.
      *                          driveId catalogs in filter will return, otherwise skip.
      * @return {Promise} driveId catalogs extended
      */
     function getDriveIdCatalogExt(nodeId, filter) {
-        var virtualDiskData, controllerData;
+        var virtualDiskData, controllerVendors = [];
```

Comment:
From the code in line 116,  the `controllerVendors` should be an object, so here need be initialized to `{}`